services/jsdelivr/jsdelivr-hits-github.service.js

```diff
@@ -0,0 +1,78 @@
+'use strict'
+
+const {
+  schema,
+  keywords,
+  periodMap,
+  BaseJsDelivrService,
+} = require('./jsdelivr-base')
+
+module.exports = class jsDelivrHitsGitHub extends BaseJsDelivrService {
+  static get route() {
+    return {
+      base: 'jsdelivr/gh',
+      pattern: ':period(hd|hw|hm|hy)/:user/:repo',
+    }
+  }
+
+  async handle({ period, user, repo }) {
+    const { total } = await this.fetch({ period, user, repo })
+    return this.constructor.render({ period, hits: total })
+  }
+
+  async fetch({ period, user, repo }) {
+    const url = `https://data.jsdelivr.com/v1/package/gh/${user}/${repo}/stats/date/${
+      periodMap[period]
+    }`
+
+    return this._requestJson({
```

Comment:
Please add a `User-Agent` header [as recommended](https://github.com/jsdelivr/data.jsdelivr.com#let-us-know-how-you-use-this-api).